### PR TITLE
Feature/timetracker

### PR DESCRIPTION
**Pull request by jonasfranz**
Adding feature tracking API implementation.

See go-gitea/gitea/pull/2211

### CODE REVIEW

**Comment by JonasFranzDEV** [comment]
Example: https://gist.github.com/JonasFranzDEV/b9ba83380e785d9f5fd7cd035a4dace8

File: gitea/issue_tracked_time.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	Created time.Time `json:"created"`
+	// Time in seconds
+	Time    int64 `json:"time"`
+	UserID  int64 `json:"user_id"`
+	IssueID int64 `json:"issue_id"`
+}
+
+// TrackedTimes represent a list of tracked times
+// swagger:response TrackedTimes
+type TrackedTimes []*TrackedTime
```
Comment (lafriks) [inline gitea/issue_tracked_time.go:27]: Why this type is needed if it is not used anywhere?

Comment (jonasfranz) [reply gitea/issue_tracked_time.go:27]: It is used for Swagger

Comment (lafriks) [reply gitea/issue_tracked_time.go:27]: Can it then be used also as return type for these functions?

Comment (jonasfranz) [reply gitea/issue_tracked_time.go:27]: @lafriks I replaced []*TrackedTime with TrackedTimes

**Comment by lafriks** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Pull request merged**